Vagrantfile

```diff
@@ -0,0 +1,10 @@
+# -*- mode: ruby -*-
```

Comment:
I'm not familiar with how Vagrant works, but I don't really want this at the top level of the repository - can we put it in `util/` (or maybe `toolchain/` is a better place...)?